### PR TITLE
Added description to eventing dashboards

### PR DESCRIPTION
**Pull request by mfaizanse**
**Description**

Changes proposed in this pull request:

- Added description to eventing dashboards
- Replaced `Total delivered messages` widget by `Pending Re-delivery Messages`, because `Total delivered messages` widget was incorrectly counting messages.
- Changed the sequence of widgets in NATS JetStream dashboard

**NATS Servers**
image

**NATS JetStream**
image

**Related issue(s)**

- https://github.com/kyma-project/kyma/issues/14134

### CODE REVIEW

File: resources/eventing/charts/dashboards/templates/jetstream.yaml [hunk 1]
```diff
@@ -101,7 +101,8 @@ data:
                    "refId":"A"
                 }
              ],
-             "title":"Storage Used",
+             "title":"File Storage Used (%)",
```
Comment (FriedrichWilken) [inline resources/eventing/charts/dashboards/templates/jetstream.yaml:104]: nit:
the dial already says "%" so I am not sure if this is needed in the description too.

File: resources/eventing/charts/dashboards/templates/jetstream.yaml [hunk 2]
```diff
@@ -219,7 +221,8 @@ data:
                    "refId":"A"
                 }
              ],
-             "title":"Memory Used",
+             "title":"In-Memory Storage Used (%)",
```
Comment (FriedrichWilken) [inline resources/eventing/charts/dashboards/templates/jetstream.yaml:224]: nit:
the dial already says "%" so I am not sure if this is needed in the description too.

File: resources/eventing/charts/dashboards/templates/jetstream.yaml [hunk 3]
```diff
@@ -1061,14 +1162,15 @@ data:
...
+                   "expr":"sum(nats_consumer_num_ack_pending{pod=~\"$server\",stream_name=~\"$stream\",consumer_desc=~\"$consumer\"}) by (consumer_desc)",
                    "hide":false,
                    "interval":"",
                    "legendFormat":"{{consumer_desc}}",
                    "refId":"A"
                 }
              ],
-             "title":"Pending messages",
+             "title":"Message Acks Pending",
+             "description": "Number of messages forwarded to the dispatcher but have not been acknowledged by the dispatcher (per consumer)",
```
Comment (FriedrichWilken) [inline resources/eventing/charts/dashboards/templates/jetstream.yaml:1173]:
```suggestion
             "description": "Number of messages forwarded to the dispatcher that have not been acknowledged by the dispatcher (per consumer)",
```

**Review by FriedrichWilken — CHANGES_REQUESTED** [review]

File: resources/eventing/charts/dashboards/templates/jetstream.yaml (hunk 1 above)
Comment (mfaizanse) [reply resources/eventing/charts/dashboards/templates/jetstream.yaml:104]: There is another tile `File Storage Used` ([here](https://github.com/kyma-project/kyma/blob/dea1020ac29bc493e47697177258e5eaac5e0297/resources/eventing/charts/dashboards/templates/jetstream.yaml#L160)) which shows the actual amount used, so to differentiate with that tile I added the % sign.

File: resources/eventing/charts/dashboards/templates/jetstream.yaml (hunk 2 above)
Comment (mfaizanse) [reply resources/eventing/charts/dashboards/templates/jetstream.yaml:224]: There is another tile `In-Memory Storage Used` ([here](https://github.com/kyma-project/kyma/blob/dea1020ac29bc493e47697177258e5eaac5e0297/resources/eventing/charts/dashboards/templates/jetstream.yaml#L280)) which shows the actual amount used, so to differentiate with that tile I added the % sign.

**Review by FriedrichWilken — APPROVED** [review]

**Pull request merged**